thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/DetectionResource.java

```diff
@@ -562,4 +564,19 @@ public Response getPredictedBaseline(
     return Response.ok(baselineTimeseries).build();
   }
 
+
+  @GET
+  @Path(value = "/health/{id}")
+  @ApiOperation("Get the detection health metrics and statuses for a detection config")
+  public Response getDetectionHealth(@PathParam("id") @ApiParam("detection config id") long id,
+      @ApiParam("Start time for the the health metric") @QueryParam("start") long start,
+      @ApiParam("End time for the the health metric") @QueryParam("end") long end,
+      @ApiParam("Max number of detection tasks returned") @QueryParam("limit") @DefaultValue("500") long limit) throws JsonProcessingException {
+    DetectionHealth health = new DetectionHealth.Builder(id, start, end).addRegressionStatus(this.evaluationDAO)
```

Comment:
Add a log statement. Consider wrapping this in try-catch similar to YamlResource.

Reply:
Added the try-catch